src/Transformations/EntityToArraySimplifier.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Consolidation\OutputFormatters\Transformations;
+
+use Consolidation\OutputFormatters\Options\FormatterOptions;
+use Consolidation\OutputFormatters\StructuredData\Xml\DomDataInterface;
+use Consolidation\OutputFormatters\StructuredData\Xml\XmlSchema;
+
+/**
+ * Simplify a EntityInterface to an array.
+ */
+class EntityToArraySimplifier implements SimplifyToArrayInterface
+{
+    public function __construct()
+    {
+    }
+
+    /**
+     * @param ReflectionClass $dataType
+     */
+    public function canSimplify(\ReflectionClass $dataType)
+    {
+        return class_exists('\Drupal\Core\Entity\EntityInterface', false) && $dataType->implementsInterface('\Drupal\Core\Entity\EntityInterface');
```

Comment:
I don't like the Drupal-specific class check here. Drush could add this simplifier in its DI setup.